core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts

```diff
@@ -147,3 +147,9 @@ export function sanitizeURL(url: string): string {
         })
         .join('/');
 }
+
+export function createIframeUrlByRenderedPage(rendered: string) {
+    const base64EncodedHtml = btoa(rendered);
```

Comment:
What is the diff between this and blob? Which one is better to our use case?

Reply:
To our use case, is better blob (Is more light and secure), i changed this function
Jest have a proble with URL object, so in some tests i add a manually global mock to this function

Reply:
TIL 
<img width="743" alt="Screenshot 2024-02-13 at 17 38 19" src="https://github.com/dotCMS/core/assets/144152756/708fcd46-50f2-4d98-be02-37160b7c8539">